packages/utils/ast-utils.js

```diff
@@ -432,6 +432,7 @@ function addProperty(j, p, key, value, action) {
 		});
 		valForNode = objectExp;
 	} else {
+		value = `'${value}'`;
```

Comment:
this is incorrect. The reason why your tests are failing on remove is because you haven't double-quoted your strings.